### ISSUE TITLE
Keyboard shortcut for checkout branch not working

### ISSUE THREAD

**Issue by ParadoxKG** [gitextensions/gitextensions#10495]
### Environment

- Git Extensions 4.0.0.15569
- Build cba315df5366fca01240d93b6e5a5077fd5e6630
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.19044.0
- .NET 6.0.11
- DPI 120dpi (125% scaling)

### Issue description

The keyboard shortcut Ctrl+. for "checkout branch" is not working. Pressing the keys has no effect. The shortcut is shown in the pull-down-menu.

Expected: Dialog "checkout branch" opens

Other shortcuts like Ctrl+Space or Ctrl+Down work as expected. Clicking the menu item for "checkout branch" works as expected.
Same behaviour both with German and English language settings for keyboard

### Steps to reproduce

- start GE and select repository
- press Ctrl+.
- => nothing happens

### Did this work in previous version of GitExtensions?

Yes, all GE 3.X

**Comment by mstv** [comment]
Duplicate of #10465
Refer to https://github.com/gitextensions/gitextensions/issues/10465#issuecomment-1329609799

**Comment by ParadoxKG** [comment]
Then I suggest changing the default to [Ctrl+.] instead of [Ctrl+NumpadDecimal].

Rationale: 
- In GE3 [Ctrl+.] did work. I was using it every day, so that is what people are used to/expect.
- [Ctrl+.] does work for all language setups. [Ctrl+NumpadDecimal] does not work for those languages where the decimal is the comma, not the dot. E.g. it is broken by default for all German keyboards.

**Comment by mstv** [comment]
> E.g. it is broken by default for all German keyboards.

Not for mine. When press it, a "," is inserted here. But `Strg + Numpad ,` triggers the checkout in GE.

**Comment by mstv** [comment]
(I guess this is a side effect of the switch from .net framework to .NET 6.)

If we change just the default, it will not have an effect for existing settings files.

@all
1. Is this change really intended by .NET?
2. Should we change the default hotkey from `Ctrl+Decimal` (at numpad) to `Ctrl+OemPeriod`?
3. Should we add migration code which adapts the existing hotkey? (It must be ensured that this code is run only once, in order to not overwrite changes by the user.)

**Comment by ParadoxKG** [comment]
> > E.g. it is broken by default for all German keyboards.
> 
> Not for mine. When press it, a "," is inserted here. But `Strg + Numpad ,` triggers the checkout in GE.

So the situation is more complicated then. :-)

In GE3 I always used Ctrl+. - I never tried the NumPad as this is labelled with "comma" on a german keyboard. So in GE3, actually both shortcuts work: Ctrl+. on main keyboard and Ctrl+NumDecimal even if that is actually a comma.

I would guess that the change in .net behaviour is not intentional - at least it is not beneficial. It would be best to get both versions to work, to maintain compatiblity with GE3.

If that is not possible, I would prefer to change it to Ctrl+. on the main keyboard.
- it is what I am used to (ok, not that important to the rest of the world :-))
- it is what matches the shortcut hint "Ctrl-." in the menu in all languages
- many laptops don't even have a numpad, so main keyboard working should have priority

**Comment by mstv** [comment]
> it is what matches the shortcut hint "Ctrl-." in the menu in all languages
> many laptops don't even have a numpad, so main keyboard working should have priority

valid points

>> 1. Is this change really intended by .NET?
>> 2. Should we change the default hotkey from `Ctrl+Decimal` (at numpad) to `Ctrl+OemPeriod`?
>> 3. Should we add migration code which adapts the existing hotkey? (It must be ensured that this code is run only once, in order to not overwrite changes by the user.)

RE 1: Do you know this, @RussKie?
RE 2: Let's do this.
RE 3: The complexity for migration code is not worth it. The users should adapt their existing settings manually.
